CP03ISSM/CP03ISSM_R00010_ingest.csv

```diff
@@ -1,43 +1,43 @@
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl36/ADCP_sn*/_RDI*.000,CP03ISSM-MFD35-01-ADCPTF000,recovered_inst,Available,
+mi.dataset.driver.presf_abc.dcl.presf_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl36/presf/*presf.log,CP03ISSM-MFD35-02-PRESFB000,recovered_host,Available,
+mi.dataset.driver.presf_abc.presf_abc_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl36/PRESF_sn*/PRESF*.hex,CP03ISSM-MFD35-02-PRESFB000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl36/velpt2/*velpt2.log,CP03ISSM-MFD35-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl36/VELPT_snAQD*/*.aqd,CP03ISSM-MFD35-04-VELPTA000,recovered_inst,Available,
+mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl36/pco2w/*pco2w.log,CP03ISSM-MFD35-05-PCO2WB000,recovered_host,Available,
+mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl36/PCO2W_snC*/SAMI*.txt,CP03ISSM-MFD35-05-PCO2WB000,recovered_inst,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl36/phsen2/*phsen2.log,CP03ISSM-MFD35-06-PHSEND000,recovered_host,Available,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl36/PHSEN_sn*/SAMI*.txt,CP03ISSM-MFD35-06-PHSEND000,recovered_inst,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl37/syslog/*.syslog.log,CP03ISSM-MFD37-00-DCLENG000,recovered_host,Available,
+#mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl37/optaa2/*.optaa2.log,CP03ISSM-MFD37-01-OPTAAD000,recovered_host,Not Deployed,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl37/ctdbp2/*ctdbp2.log,CP03ISSM-MFD37-03-CTDBPD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl37/CTDBP_sn*/SBE*.hex,CP03ISSM-MFD37-03-CTDBPD000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl37/dosta2/*dosta2.log,CP03ISSM-MFD37-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/cg_data/dcl37/zplsc/*zplsc.log,CP03ISSM-MFD37-07-ZPLSCC000,recovered_host,Available,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00010/instruments/dcl37/ZPLSC*/DATA/*/*.01A,CP03ISSM-MFD37-07-ZPLSCC000,recovered_inst,Available,Test parser

```

Comment:
Commented out this line because we cannot correctly ingest ZPLSC recovered instrument data yet. fixed filemask so it could find data